tasks/GuildShardDonation/script_task.py

```
@@ -0,0 +1,340 @@
+        try:
+            for round_index in range(self.MAX_SCAN_ROUNDS):
+                logger.hr(f"碎片捐赠扫描 {round_index + 1}/{self.MAX_SCAN_ROUNDS}", level=1)
+                self._enter_prayer_page()
+                target = self._scan_for_player()
+                if target is None:
+                    logger.warning("本轮完整扫描未找到目标玩家：%s", self.target_player)
+                    self._return_to_main()
+                    continue
+
+                if not self._click_give_on_same_row(target):
+                    logger.info("目标玩家同行没有‘赠予’按钮，按成功结束")
```

Ruff (0.15.21)

[warning] String contains ambiguous `‘` (LEFT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)